.gitlab/package_build/linux.yml

```
@@ -54,29 +54,46 @@
     DD_CXX: "aarch64-unknown-linux-gnu-g++"
     DD_CMAKE_TOOLCHAIN: "/opt/cmake/aarch64-unknown-linux-gnu.toolchain.cmake"
 
-.agent_7_build:
+.agent_base_flavor:
   variables:
     FLAVOR: base
 
-.agent_fips_build:
+.agent_fips_flavor:
   variables:
     FLAVOR: fips
 
 # build Agent 7 binaries for x86_64
 datadog-agent-7-x64:
-  extends: [.agent_build_common, .agent_build_x86, .agent_7_build]
+  extends: [.agent_build_common, .agent_build_x86, .agent_base_flavor]
 
 # build Agent 7 binaries for arm64
 datadog-agent-7-arm64:
-  extends: [.agent_build_common, .agent_build_arm64, .agent_7_build]
+  extends: [.agent_build_common, .agent_build_arm64, .agent_base_flavor]
 
 # build Agent 7 binaries for x86_64 with FIPS
 datadog-agent-7-x64-fips:
-  extends: [.agent_build_common, .agent_build_x86, .agent_fips_build]
+  extends: [.agent_build_common, .agent_build_x86, .agent_fips_flavor]
 
 # build Agent 7 binaries for arm64 with FIPS
 datadog-agent-7-arm64-fips:
-  extends: [.agent_build_common, .agent_build_arm64, .agent_fips_build]
+  extends: [.agent_build_common, .agent_build_arm64, .agent_fips_flavor]
+
+# Common to IoT, DDOT and dogstatsd
+.toolchain-common-x64:
```

`x64` is as misnomer - I realize Microsoft's marketing has already invaded the repo, but let's stick to either the more conventional `amd64`:
```suggestion
.toolchain-common-amd64:
```
... or semantically correct `x86_64`:
```suggestion
.toolchain-common-x86_64:
```

---

> I can do a follow-up PR that normalizes the arch names ?

Any contribution to sorting this out would be more than welcome, but we can't in all fairness ask you to devote time to it.

> The other agent jobs mainly use `x64` for amd64

bad karma?

> the UEFI bootloader naming convention uses `X64`, so not just MS marketing

[chicken](https://learn.microsoft.com/en-us/windows/win32/debug/pe-format#:~:text=IMAGE_FILE_MACHINE_AMD64,x64) and [egg](https://uefi.org/specs/UEFI/2.11/02_Overview.html#:~:text=EFI_IMAGE_MACHINE_x64%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%200x8664)?
> | Constant | Value | Description |
> | - | - | - |
> | IMAGE_FILE_MACHINE_AMD64 | **0x8664** | x64 |
```c
// PE32+ Machine type for EFI images
#define EFI_IMAGE_MACHINE_x64                        0x8664
```